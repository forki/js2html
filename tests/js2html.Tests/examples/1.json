{
    "@id": "http://ld.nice.org.uk/prov/entity#98ead3d:qualitystandards/qs70/st2/Statement.md",
    "@type": ["prov:Entity", "owl:NamedIndividual"],
    "qualitystandard:setting": "http://ld.nice.org.uk/ns/qualitystandard/setting#Primary care setting",
    "qualitystandard:targetPopulation": ["http://ld.nice.org.uk/ns/qualitystandard/agegroup#Children 1-15 years", "http://ld.nice.org.uk/ns/qualitystandard/agegroup#Young people", "http://ld.nice.org.uk/ns/qualitystandard/conditiondisease#Urinary incontinence"],
    "dcterms:abstract": "Children and young people have an agreed review date if they, or their\nparents or carers, are given advice about changing their daily routine\nto help with bedwetting.",
    "dcterms:title": "Quality statement 2: Review after initial advice is given",
    "prov:specializationOf": "resource:FF2ED3E529E5BDCDED86C24D3277F8A156814492DFE352A09804DF3228109D63",
    "_id": "resource:FF2ED3E529E5BDCDED86C24D3277F8A156814492DFE352A09804DF3228109D63",
    "_type": "qualitystatement"
}
