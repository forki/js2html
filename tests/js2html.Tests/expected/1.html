<div>http://ld.nice.org.uk/prov/entity#98ead3d:qualitystandards/qs70/st2/Statement.md</div>
<div>resource:FF2ED3E529E5BDCDED86C24D3277F8A156814492DFE352A09804DF3228109D63</div>
<div>http://ld.nice.org.uk/ns/qualitystandard/setting#Primary care setting</div>
<div>Children and young people have an agreed review date if they, or their
parents or carers, are given advice about changing their daily routine
to help with bedwetting.</div>
<ul>
    <li>prov:Entity</li>
    <li>owl:NamedIndividual</li>
</ul>
